cmd/clusterctl/validation/validate_cluster_api_objects_test.go

```diff
@@ -76,34 +74,38 @@ func getNodeWithReadyStatus(nodeName string, nodeReadyStatus v1.ConditionStatus)
 }
 
 func TestGetClusterObjectWithNoCluster(t *testing.T) {
-	g := gomega.NewGomegaWithT(t)
-
 	// Setup the Manager and Controller.
 	mgr, err := manager.New(cfg, manager.Options{})
-	g.Expect(err).NotTo(gomega.HaveOccurred())
+	if err != nil {
+		t.Fatalf("error creating new manager: %v", err)
+	}
 	c = mgr.GetClient()
-	defer close(StartTestManager(mgr, g))
+	defer close(StartTestManager(mgr, t))
 
 	_, err = getClusterObject(c, "test-cluster", "get-cluster-object-with-no-cluster")
-	g.Expect(err).To(gomega.HaveOccurred())
+	if err == nil {
+		t.Error("expected error but didn't get one")
+	}
 }
 
 func TestGetClusterObjectWithOneCluster(t *testing.T) {
-	g := gomega.NewGomegaWithT(t)
-
 	// Setup the Manager and Controller.
 	mgr, err := manager.New(cfg, manager.Options{})
-	g.Expect(err).NotTo(gomega.HaveOccurred())
+	if err != nil {
+		t.Fatalf("error creating new manager: %v", err)
+	}
 	c = mgr.GetClient()
-	defer close(StartTestManager(mgr, g))
+	defer close(StartTestManager(mgr, t))
 
 	const testClusterName = "test-cluster"
 	const testNamespace = "get-cluster-object-with-one-cluster"
 	cluster := testutil.GetVanillaCluster()
 	cluster.Name = testClusterName
 	cluster.Namespace = testNamespace
 	err = c.Create(context.TODO(), &cluster)
-	g.Expect(err).NotTo(gomega.HaveOccurred())
+	if err != nil {
```

Comment:
Cases like this should be cuddled

`if err := c.Create(...); err != nil { ... }`